### PR TITLE
Clean dialogs 

### PR DESCRIPTION
**Pull request by justend29**
- cleans input dialogs a bit
- doesn't let managers in when the create org/ create org manager fails

Check commit messages from March22 for log, they explain

### CODE REVIEW

**Review by StephenCole19 — CHANGES_REQUESTED** [review]
Besides minor request looks good 👍

**Comment by StephenCole19** [comment]
I resolved some conflicts between @ChrisRuff s branch and yours. @justend29 if you don't mind re-testing this change, if I messed anything up feel free to revert the last commit and resolve the conflicts properly.

File: lib/views/homepage.dart [hunk 1]
```diff
@@ -107,30 +119,35 @@ class _HomePageState extends State<HomePage> {
...
-                  onChanged: (String? value) {
-                    setState(() {
-                      _enableExamination = value != null && value.isNotEmpty;
-                    });
-                  }),
+              Padding(
+                  padding: const EdgeInsets.fromLTRB(0,0,40,0),
+                  child: TextFormField(
+                      textAlign: TextAlign.center,
+                      //keyboardType: TextInputType.numberWithOptions(decimal: true),
```
[the remark this answers is not in the document]
Comment (justend29) [reply lib/views/homepage.dart:126]: thk u

File: lib/widgets/registerorgform.dart [hunk 2]
```diff
@@ -45,7 +46,7 @@ class _RegisterOrgFormState extends State<RegisterOrgForm> {
     return false;
   }
 
-  // TODO: Chris, why are you returning empty strings here??
+  // TODO: this provides no error message or reason. Create nice message, like registerOfficer()
```
Comment (StephenCole19) [inline lib/widgets/registerorgform.dart:49]: Remove todo comment instead of communicating through them.

File: lib/widgets/registerorgform.dart [hunk 3]
```diff
@@ -127,69 +130,75 @@ class _RegisterOrgFormState extends State<RegisterOrgForm> {
   // maybe a scrollable form?
```
Comment (StephenCole19) [inline lib/widgets/registerorgform.dart:130]: Is this comment still relevant?

File: lib/views/homepage.dart [hunk 4]
```diff
@@ -24,6 +24,18 @@ class HomePage extends StatefulWidget {
   State<HomePage> createState() => _HomePageState();
 }
 
+// https://stackoverflow.com/questions/49238908/flutter-textfield-value-always-uppercase-debounce
```
Comment (StephenCole19) [inline lib/views/homepage.dart:27]: Maybe add to link to docs instead.

**Review by StephenCole19 — CHANGES_REQUESTED** [review]
Just remove a few of those comments and it looks good to me.

**Comment by StephenCole19** [comment]
@ChrisRuff

**Review by ChrisRuff — APPROVED** [review]

File: lib/views/homepage.dart [hunk 5]
```diff
@@ -211,15 +232,21 @@ class _HomePageState extends State<HomePage> {
...
                     ],
                   ),
-                )
+                  secondChild: const SizedBox(),
+                  crossFadeState: !isKeyboardVisible
+                      ? CrossFadeState.showFirst
+                      : CrossFadeState.showSecond),
+              if (!isKeyboardVisible) const Spacer(flex: 2)
+              // if (!isKeyboardVisible)
+              //   const LimitedBox(maxHeight: 50, child: Spacer())
```
Comment (StephenCole19) [inline lib/views/homepage.dart:241]: ^

**Review by StephenCole19 — CHANGES_REQUESTED** [review]

File: lib/views/homepage.dart (hunk 5 above)
Comment (justend29) [reply lib/views/homepage.dart]: ahh, i see there is more

**Review by StephenCole19 — APPROVED** [review]

**Pull request merged**